Vungle/src/main/java/com/mopub/mobileads/VungleRewardedVideo.java

```diff
@@ -204,11 +203,17 @@ private boolean validateIdsInServerExtras(Map<String, String> serverExtras) {
     }
 
     private void setUpMediationSettingsForRequest(AdConfig adConfig) {
-        final VungleMediationSettings globalMediationSettings =
-                MoPubRewardedVideoManager.getGlobalMediationSettings(VungleMediationSettings.class);
-        final VungleMediationSettings instanceMediationSettings =
-                MoPubRewardedVideoManager.getInstanceMediationSettings(VungleMediationSettings.class, mAdUnitId);
-
+         VungleMediationConfiguration globalMediationSettings =
+                MoPubRewardedVideoManager.getGlobalMediationSettings(VungleMediationConfiguration.class);
```

Comment:
For api `getGlobalMediationSettings` and `getInstanceMediationSettings` MoPub only handle the two classes which must be of exactly equal types.